README.md

```diff
@@ -78,6 +81,56 @@ loadPerEnv({path: './feature.js', identifier: 'feature'}, 'NODE_ENV', "dev");
 import * as feature from "./feature.js";
 ```
 
+### `loadPerEnvMap`
+
+Allows you to use a map of different imports based on the value of your envars
+
+```javascript
+import { loadPerEnvMap } from "./perEnv.macro";
+loadPerEnvMap({dev: './new_feature.js', production: './feature.js'}, 'NODE_ENV');
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+import "./feature.js";
+```
+
+Use an assignment to declare the import namespace, handy to keep linters quiet
+
+```javascript
+import { loadPerEnvMap } from "./perEnv.macro";
+const feature = loadPerEnvMap({dev: './new_feature.js', production: './feature.js'}, 'NODE_ENV');
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+import * as feature from "./feature.js";
+```
+
+Enable the `nullable` flag to allow for cases where the envar value may not be in the map.
+
+
+```javascript
+
+// NODE_ENV=CI
+import { loadPerEnvMap } from "./perEnv.macro";
+const feature = loadPerEnvMap({dev: './new_feature.js', production: './feature.js'}, 'NODE_ENV', true);
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+const feature = null
+```
+
+Or in the case that no assignment is made
+
+```javascript
+
+// NODE_ENV=CI
+import { loadPerEnvMap } from "./perEnv.macro";
+const feature = loadPerEnvMap({dev: './new_feature.js', production: './feature.js'}, 'NODE_ENV', true);
```

Comment:
Shouldn't this example have no assignment?

Reply:
```suggestion
// NODE_ENV=  (case where NODE_ENV is unset)
const feature = loadPerEnvMap({dev: './new_feature.js', production: './feature.js'}, 'NODE_ENV', true);
```

Reply:
I've made changes to the readme, if you feel it's clear enough, I'll resolve this.